### PR TITLE
Show notes for episode 3

### PR DESCRIPTION
**Pull request by jakelacey2012**

### CODE REVIEW

File: 003-my-dad-got-sextorted.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# My Dad Got Sextorted
+
+----
```
Comment (jakelacey2012) [inline 003-my-dad-got-sextorted.md:3]:
```suggestion
In this episode we talk about how scammers can scare you into doing what they want.
```

File: 003-my-dad-got-sextorted.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## What have we been working on?
+
+### Greg's week
+
+(00:24)
+
++ Pledge never touch HTML emails again, but having to touch them.
++ Hidden complexities of HTML email.
++ Setting upå servers for clients, linux tooling giving quick wins
```
Comment (g105b) [inline 003-my-dad-got-sextorted.md:15]: å?

Comment (g105b) [reply 003-my-dad-got-sextorted.md:15]: While I'm reviewing, can I be picky and say this is the only line that doesn't end with a full stop?

**Review by g105b — CHANGES_REQUESTED** [review]

File: 003-my-dad-got-sextorted.md (hunk 2 above)
Comment (jakelacey2012) [reply 003-my-dad-got-sextorted.md:15]: Yes you can ❤️

**Pull request merged**